typescript/turbo.json

```diff
@@ -4,7 +4,12 @@
   "tasks": {
     "build": {
       "dependsOn": ["^build"],
-      "outputs": ["dist/**"]
+      "outputs": [
+        "dist/**",
+        ".next/**",
+        "!.next/cache/**",
```

Comment:
Fixes warnings from templates that get added to the workspace, but whose build process differs from how we've been building to `/dist`. Specifically, the Next and Mcp Server templates require this to get rid of the cli warnings on build

Reply:
Should we even be building the templates as part of our normal build process? If/when the templates incorporate nunjucks, the builds will stop working

Reply:
(not blocking, can be a follow-up / still up for debate)

Reply:
True, right now we're just building the projects that don't use nunjucks. Next was not affected by that PR, I just used it for the new generate commands.

I am fine with straight up ignoring them in a follow